lib/connect_four.rb

```diff
@@ -156,35 +156,46 @@ def valid_column(input)
         end
     end
 
-    #we can refactor valid_column, row_index, and column_index into one method
+    # we can refactor valid_column, row_index, and column_index into one method
```

Comment:
I think I know what you mean but I don't fully understand how to make it one method